Traceback (most recent call last):
  File "/home/spi085/Documents/_Projects/EMs-with-LLMs/POPULATIONS/POPULATION_0011/CULLED/INDIVIDUAL_9EL59F6A/form_check/plot_forms_NPZ.py", line 233, in <module>
    main()
  File "/home/spi085/Documents/_Projects/EMs-with-LLMs/POPULATIONS/POPULATION_0011/CULLED/INDIVIDUAL_9EL59F6A/form_check/plot_forms_NPZ.py", line 174, in main
    _savefig(fig1, f"CULLED_INDIVIDUAL_9EL59F6A_npz_states", SAVE_DIR, SAVE_FMT, MODEL_ID, dpi=300)
TypeError: _savefig() got an unexpected keyword argument 'dpi'
